src/shared/cache-types.cts

```diff
@@ -154,3 +158,60 @@ export type CacheHandlerForMultipleVersions = BaseCacheHandlerForMultipleVersion
     context: CacheHandlerSetContextForMultipleVersions,
   ) => ReturnType<BaseCacheHandlerForMultipleVersions['set']>
 }
+
+export type TagManifest = { revalidatedAt: number }
+
+export type HtmlBlob = {
+  html: string
+  isFallback: boolean
+}
+
+export type BlobType = NetlifyCacheHandlerValue | TagManifest | HtmlBlob
+
+const isTagManifest = (value: BlobType): value is TagManifest => {
+  return false
+}
+
+const isHtmlBlob = (value: BlobType): value is HtmlBlob => {
+  return false
+}
+
+export const estimateBlobSize = (valueToStore: BlobType | null | Promise<unknown>): number => {
```

Comment:
Same here, this is not a "cache type" so I think we should put this in a different file

Reply:
See https://github.com/opennextjs/opennextjs-netlify/pull/2777#discussion_r2016264366